pkgs/development/tools/misc/wishbone-tool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, rustPlatform, libusb }:
+let
+  version = "0.2.8";
+  src = fetchFromGitHub {
+    owner = "xobs";
+    repo = "wishbone-utils";
+    rev = "v${version}";
+    sha256 = "1ck3y55x3zn5qm4m65m678p0z800cjscmn9dvpadzk13d22vvjrj";
+  };
+in
+rustPlatform.buildRustPackage {
+  pname = "wishbone-tool";
+  inherit version;
+  src = "${src}/wishbone-tool";
+  cargoSha256 = "0pj8kf6s1c666p4kc6q1hlvaqm0lm9aqnsx5r034g1y8sxnnyri2";
+  buildInputs = [ libusb ];
+
+  # as of 0.2.8, the .cargo/config is missing a final newline
+  preConfigure = "echo >> .cargo/config";
```

Comment:
Is that actually necessary? I don't seem to see any issues after removing this locally.